docs/testplan/DPU-test-plan.md

```diff
@@ -0,0 +1,863 @@
+      ------------  --------  -------------------  ------------  --------------  -------------
+      Ethernet-BP0            18.0.202.0/31        up/up         N/A             N/A
+      Ethernet-BP1            18.1.202.0/31        up/up         N/A             N/A
+      Ethernet-BP2            18.2.202.0/31        up/up         N/A             N/A
+      Ethernet-BP3            18.3.202.0/31        up/up         N/A             N/A
+      Ethernet-BP4            18.4.202.0/31        up/up         N/A             N/A
+      Ethernet-BP5            18.5.202.0/31        up/up         N/A             N/A
+      Ethernet-BP6            18.6.202.0/31        up/up         N/A             N/A
+      Ethernet-BP7            18.7.202.0/31        up/up         N/A             N/A
+      docker0                 240.127.1.1/24       up/down       N/A             N/A
+      eth0                    172.25.42.65/24      up/up         N/A             N/A
+      eth1                    169.254.24.2/24      up/up         N/A             N/A
+      eth2                    169.254.28.2/24      up/up         N/A             N/A
+      eth3                    169.254.32.2/24      up/up         N/A             N/A
+      eth4                    169.254.36.2/24      up/up         N/A             N/A
+      eth5                    169.254.139.2/24     up/up         N/A             N/A
+      eth6                    169.254.143.2/24     up/up         N/A             N/A
+      eth7                    169.254.147.2/24     up/up         N/A             N/A
+      eth8                    169.254.151.2/24     up/up         N/A             N/A
+      lo                      127.0.0.1/16         up/up         N/A             N/A
+      root@sonic:/home/cisco# 
+```
+#### Pass/Fail Criteria
+ * Verify output on switch to see all 169.254.x.x networks are showing up.
+
+### 1.3 Check fan LED speed descripton and presence
+
+#### Steps
+ * Use command `show platform fan` to get FAN speed and Presence
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:/home/cisco# show platform fan
+  Drawer    LED            FAN    Speed    Direction    Presence    Status          Timestamp
+--------  -----  -------------  -------  -----------  ----------  --------  -----------------
+     N/A    N/A      PSU0.fan0      50%          N/A     Present        OK  20230907 07:04:55
+     N/A    N/A      PSU1.fan0      50%          N/A     Present        OK  20230907 07:04:55
+fantray0    N/A  fantray0.fan0      52%       intake     Present        OK  20230907 07:04:55
+fantray1    N/A  fantray1.fan0      52%       intake     Present        OK  20230907 07:04:55
+fantray2    N/A  fantray2.fan0      52%       intake     Present        OK  20230907 07:04:55
+fantray3    N/A  fantray3.fan0      52%       intake     Present        OK  20230907 07:04:55
+root@sonic:/home/cisco#
+
+```
+#### Pass/Fail Criteria
+ * Verfiy Presence, LED (green) and speed on the output
```

Comment:
typo

Reply:
Speed is dependent on outside circumstances, hard to have as a pass criteria. Would be good to clarify that it just shouldn't be `N/A`. As for LEDs, we don't have them yet but hopefully soon :)

Reply:
Corrected the typo. 
Yes we can generally check the status, presence as of now and go ahead with LED once its implemented.